core/blockchain.go

```diff
@@ -1693,6 +1693,44 @@ func (bc *BlockChain) InsertChain(chain types.Blocks) (int, error) {
 	return n, err
 }
 
+// InsertChainWithoutSealVerification works exactly the same
+// except for seal verification, seal verification is omitted
+func (bc *BlockChain) InsertChainWithoutSealVerification(chain types.Blocks) (int, error) {
```

Comment:
This is ever called with a slice containing one block, AFAICT. Might just make it take a single block on input, and throw away the whole sanity check loop